app/boards/shields/eek/Kconfig.defconfig

```diff
@@ -0,0 +1,6 @@
+if SHIELD_EEK
+
+config ZMK_KEYBOARD_NAME
+    default "zmk~eek!"
```

Comment:
To be consistent with the other shields in the repo, the keyboard name shouldn't contain "zmk".

Maybe also capitalize it to match this? https://www.cbkbd.com/product/eek-keyboard